### PR TITLE
added Openshift required securityContext PodSpec

### PR DESCRIPTION
**Pull request by tsebastiani**
## Changes introduced with this PR

Added to the pod schema securityContext parameters required by OpenShift  restricted:latest security policies

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: schema.go [hunk 1]
```diff
@@ -525,9 +525,35 @@ var containerImagePullPolicyProperty = schema.NewPropertySchema(
 		},
 	),
 	schema.NewDisplayValue(
-		schema.PointerTo("Volume device"),
+		schema.PointerTo("Image Pull Policy"),
 		schema.PointerTo(
-			"Mount a raw block device within the container.",
+			"describes a policy for if/when to pull a container image",
```
Comment (tsebastiani) [inline schema.go:530]: Seems to me that there was a typo (maybe copy/paste oversight) on this enum.

File: schema.go [hunk 2]
```diff
@@ -602,28 +841,12 @@ var containerSecurityContextProperty = schema.NewPropertySchema(
...
-				nil,
-			),
 		},
 	),
 	schema.NewDisplayValue(
-		schema.PointerTo("Volume device"),
+		schema.PointerTo("Security Context"),
 		schema.PointerTo(
-			"Mount a raw block device within the container.",
+			"SecurityContext holds security configuration that will be applied to a container.",
```
Comment (tsebastiani) [inline schema.go:849]: same as imagePullPolicy mentioned above

Comment (dustinblack) [reply schema.go:849]: :+1:

**Review by dustinblack — APPROVED** [review]
LGTM

**Review by jaredoconnell — APPROVED** [review]
Looks good to me.

**Review by jdowni000 — APPROVED** [review]
lgtm

**Pull request merged**